cli/jobs/single-step/r/accidents/src/accidents.R

```diff
@@ -1,4 +1,11 @@
 library(optparse)
+library("carrier")
+source('aml.R')
```

Comment:
can you add a comment about why we're importing the aml util file here?

Reply:
please do the same for the other example as well

Reply:
Added a comment. Please review.